code/lightna/magento-os-frontend/js/common/AddToCart.js

```diff
@@ -1,4 +1,5 @@
 import { UserInput } from 'lightna/lightna-engine/lib/UserInput';
+import { PageMessage } from './PageMessage';
```

Comment:
we need to check if "./PageMessages" and "lightna/magento-os-frontend/PageMessages" makes difference for extends

Reply:
it does, by alias replaced either `./PageMessages` or `lightna/magento-os-frontend/PageMessages`; depends how it's imported; so better not to use relative path like above

Reply:
ok, lets merge it and fire new ticket to change all to full path

Reply:
no need, we don't have such cases
(this is outdated thread)